### ISSUE TITLE
Can't assess the net.minecraft, cpw package

### ISSUE THREAD

**Issue by Hyperatox** [LegacyModdingMC/examplemod#1]
#### Description
Can't assess the net.minecraft, cpw package 
(error: package net.minecraft.init does not exist)

#### Environment
java -version
openjdk version "1.8.0_382"
OpenJDK Runtime Environment (Zulu 8.72.0.17-CA-win64) (build 1.8.0_382-b05)      
OpenJDK 64-Bit Server VM (Zulu 8.72.0.17-CA-win64) (build 25.382-b05, mixed mode)

IDE: VSCode/VSCodium with: all the recommended plugins mentioned in the starting guide

#### Screenshots
![Capture](https://github.com/LegacyModdingMC/examplemod/assets/26015650/4efec14f-3493-48b7-bffc-eb0506f421a2)

#### To Reproduce
1 Clone a clean template.
2 Following the README
3 Opening the Project in VSCode

#### To Fix the issue, I tried:
	run the gradletasks in all projects/subprojects:
	- [BUILD SUCCESSFUL] unimined/GenGources
	- [BUILD SUCCESSFUL] buildsetup/init

#### Investigation
	In search of the missing Minecraft classes, I found empty Libraries
	(minecraft-1.7.10-fabric-merged+fixed-*.jar in the gradle cache)

**Comment by makamys** [comment]
What happens if you try running the `build` task?

**Comment by Hyperatox** [comment]
It fails for the same reason.
It can't find the package net.minecraft.client.gui or net.minecraft.init.Items;
https://pastebin.com/nw2F7TYJ

**Comment by makamys** [comment]
It's not an ideal solution but you could try deleting your Gradle cache (or rather, renaming it to something else so you can restore it later) and seeing if the issue persists. On Windows it can be found at `C:\Users\<user>\.gradle`.

Note that you cannot do this while the Gradle daemon is running. The easiest way to kill it is to close your IDE and kill all `java.exe`s in the task manager. Or just restart your computer.

**Comment by Hyperatox** [comment]
Leads to the same problem.
I also configured Gradle for Java 1.8, but it stays at java17 in "configure runtimes for projects"
![image](https://github.com/LegacyModdingMC/examplemod/assets/26015650/7ff7fb41-f5e3-447b-91df-5f1c70c39a59)
plus, it doesn't use $java_home at all.

**Comment by Hyperatox** [comment]
I am now able to successfully start the client task [unimined_runs/runClient], but the package issue still persists.

**Comment by Hyperatox** [comment]
I found out that VSCode still uses Java 17 after several configurations attempts.
Means my settings in VSCode did not completely prevent the use of Java 17.

My Settings:
![image](https://github.com/LegacyModdingMC/examplemod/assets/26015650/477bd0f7-ef2a-4c37-b417-11cb4e690510)

Where is my Mistake?

After several hours, I uninstalled Java 17 and was able to start the example project with Java 8 alone flawlessly.
I also Reinstalled Java 17 and VSCode continues to work, I finally can code my mods.

**Thank you, @makamys, for your help.
I apologize, I didn't realize it was my or the IDE's fault.**
